datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -539,7 +539,7 @@ private void validateMediumDutySdxEnablement(SdxClusterRequest sdxClusterRequest
         ValidationResultBuilder validationBuilder = new ValidationResultBuilder();
         if (SdxClusterShape.MEDIUM_DUTY_HA.equals(sdxClusterRequest.getClusterShape())) {
             boolean mediumDutySdxEntitlementEnabled = entitlementService.mediumDutySdxEnabled(Crn.safeFromString(environment.getCreator()).getAccountId());
-            if (!mediumDutySdxEntitlementEnabled) {
+            if (!(AWS.equalsIgnoreCase(environment.getCloudPlatform()) || mediumDutySdxEntitlementEnabled)) {
```

Comment:
I prefer not negating under brackets. For example, instead of saying "not (A or B)", you can say "not A and not B". They are logically equivalent. The reason is that the next time someone goes to modify this line, they might miss the brackets and might accidentally break the logic. We want to make it easy for the next person to not make accidental mistakes. 

To make this logic cleaner, do something like this

```
  boolean isAWS = AWS.equalsIgnoreCase(environment.getCloudPlatform());
  if (!isAWS && mediumDutySdxEntitlementEnabled) {
    // print error
  } 
```

The logic reads much more cleaner now: "If its not AWS and you have medium duty enabled, print the error"